code/modules/atmospherics/gasmixtures/gas_types.dm

```diff
@@ -19,8 +19,36 @@ GLOBAL_LIST_INIT(nonreactive_gases, typecacheof(list(/datum/gas/oxygen, /datum/g
 		gas_info[META_GAS_FUSION_POWER] = initial(gas.fusion_power)
 		gas_info[META_GAS_DANGER] = initial(gas.dangerous)
 		gas_info[META_GAS_ID] = initial(gas.id)
+
+		gas_info[META_GAS_COND_RATE] = initial(gas.cond_rate)
+		gas_info[META_GAS_COND_TEMP_MAX] = initial(gas.cond_temp_max)
+		gas_info[META_GAS_COND_TEMP_MIN] = initial(gas.cond_temp_min)
+		gas_info[META_GAS_COND_TYPE] = initial(gas.cond_type)
+		gas_info[META_GAS_COND_HEAT] = initial(gas.cond_heat)
+
+		if(GLOB.gas_singletons) // We do this twice to make sure it gets in regardless of which order these are instantiated
+			var/datum/gas/gas_singleton = GLOB.gas_singletons[gas_path]
+			gas_singleton.meta_gas_list = gas_info
+			gas_info[META_GAS_COND_EVENT] = gas_singleton.cond_event
+
 		.[gas_path] = gas_info
 
+	if(SSair)
+		SSair.gas_metadata = .
```

Comment:
Why

Reply:
It's easier to access when debugging

Reply:
cause of the pain of sorting through GLOB vs SSair?